portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/UserEntry.scala

```diff
@@ -31,4 +31,12 @@ class UserEntry(val pipelineDatabaseEngine: PipelineDatabaseEngine) {
 
   def addUserAction(user: User) = (users.returning(users.map(_.userId))) += user
 
+  def updateUserById(updatedUser: User) =
```

Comment:
I suppose, this method should be named `updateUser`. We pass whole user as a parameter

Reply:
done